lib/tasks/migrations.rake

```diff
@@ -17,22 +18,74 @@ namespace :db do
     system(%(pg_dump --exclude-table=ar_internal_metadata --no-owner -h #{db_config['host']} -p #{db_config['port']} -U #{db_config['username']} #{db_config['database']} -f db.sql))
 
     # Move old db.sql on S3 to archive folder
-    system("s3cmd --access_key=#{ENV['AWS_ACCESS_KEY']} --secret_key=#{ENV['AWS_SECRET_KEY']} mv #{BACKUP_S3_PATH}/#{VERSION}-db.sql #{ARCHIVE_S3_PATH}/#{VERSION}/#{Time.now.to_i}-db.sql")
+    system("#{S3CMD} mv #{BACKUP_S3_PATH}/#{VERSION}-db.sql #{ARCHIVE_S3_PATH}/#{VERSION}/#{Time.now.to_i}-db.sql")
```

Comment:
Let's change this to another `put` of the local `db.sql`, since the file here you're moving doesn't actually represent the backup at `Time.now`.

That way essentially every time we backup we place that sql both as the current dump (overwrite) and as a time-labeled archive file.  That's currently the behavior that's implemented, just in a separate place ([resource repo](https://github.com/Factual/resources/blob/master/services/front/kudos/backup-db.sh)).  But I will change that to use this rake task.

Also, let's change `Time.now.to_i` to `Time.now.utc.iso8601` to be consistent with what's currently stored in S3.